Packs/Safewalk/Integrations/SafewalkManagement/SafewalkManagement.yml

```diff
@@ -0,0 +1,449 @@
+commonfields:
+  id: Safewalk_Management
+  version: -1
+name: Safewalk_Management
+display: Safewalk_Management
+category: Authentication
+description: Safewalk server integration
+detaileddescription: |2
+   ### Safewalk XSOAR Integration
+   #### Integration Author: Altipeak
+configuration:
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: first_fetch
+  defaultvalue: 7 days
+  required: false
+  type: 0
+- display: Max fetch 
+  defaultvalue: '50'
+  name: max_fetch
+  required: false
+  type: 0
+- display: Server URL (e.g. https://soar.monstersofhack.com)
+  name: url
+  defaultvalue: https://safewalk-server.company.com:8443/
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  type: 13
+  required: false
+- display: API Key
+  name: apikey
+  defaultvalue: f8ee1ee7135d547368122410e19436d47fc351dc
+  type: 4
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+- display: Fetch indicators
+  name: feed
+  type: 8
+  required: false
+- display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  defaultvalue: "1"
+  type: 19
+  required: false
+- name: feedReputation
+  display: Indicator Reputation
+  type: 18
+  required: false
+  options:
+  - None
+  - Good
+  - Suspicious
+  - Bad
+  additionalinfo: Indicators from this integration instance will be marked with this
+    reputation
+- name: feedReliability
+  display: Source Reliability
+  type: 15
+  required: true
+  options:
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  additionalinfo: Reliability of the source providing the intelligence data
+- name: feedExpirationPolicy
+  display: ''
+  type: 17
+  required: false
+  options:
+  - never
+  - interval
+  - indicatorType
+  - suddenDeath
+- name: feedExpirationInterval
+  display: ''
+  type: 1
+  required: false
+- name: feedFetchInterval
+  display: Feed Fetch Interval
+  type: 19
+  required: false
+- name: feedBypassExclusionList
+  display: Bypass exclusion list
+  type: 8
+  required: false
+  additionalinfo: When selected, the exclusion list is ignored for indicators from
+    this feed. This means that if an indicator from this feed is on the exclusion
+    list, the indicator might still be added to the system.
+- name: feedTags
+  display: Tags
+  required: false
+  type: 0
+  additionalinfo: Supports CSV values.
+- name: tlp_color
+  display: Traffic Light Protocol Color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+  additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+    fetched from the feed
+script:
+  commands:
```

Comment:
Please add arguments and outputs for each of the commans.
See more info about arguments here: https://pull-request-440--demisto-content-docs.netlify.app/docs/integrations/yaml-file#commands
See more info about outputs here: https://pull-request-440--demisto-content-docs.netlify.app/docs/integrations/context-and-outputs#integration-yml